WcaOnRails/app/webpacker/components/CountrySelector/CountrySelector.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Form } from 'semantic-ui-react';
+
+import countries from '../../lib/wca-data/countries.js.erb';
+import CountryFlag from '../wca/CountryFlag';
+import '../../stylesheets/country_selector.scss';
+
+const countryOptions = countries.real.map((country) => ({
+  key: country.iso2,
+  text: country.name,
+  value: country.iso2,
+  image: <CountryFlag iso2={country.iso2} />,
+}));
```

Comment:
Ok I've just seen your other comment below referring to this one and I see your point now: I'm a bit hesitant to move `countryOptions` to `lib` because it's actually specific to rendering that specific component and I doubt any other one would require that exact format.
And if they did they'd likely want to display some kind of country selection component and should probably re-use this one.

Reply:
Okay, reusing the `CountrySelector` overall and deliberately _not_ exposing the internal representation for the dropdown choices makes sense :)